gocd/pipelines/snuba-stable.yaml

```
@@ -72,34 +72,52 @@ pipelines:
                                     && /devinfra/scripts/k8s/k8s-deploy.py \
                                     --label-selector="service=snuba,is_canary=true" \
                                     --image="us-central1-docker.pkg.dev/sentryio/snuba/image:${GO_REVISION_SNUBA_REPO}" \
-                                    --container-name="api" \
-                                    --container-name="consumer" \
-                                    --container-name="errors-consumer" \
-                                    --container-name="errors-replacer" \
-                                    --container-name="events-subscriptions-executor" \
-                                    --container-name="events-subscriptions-scheduler" \
-                                    --container-name="generic-metrics-counters-consumer" \
-                                    --container-name="generic-metrics-counters-subscriptions-executor" \
-                                    --container-name="generic-metrics-counters-subscriptions-scheduler" \
-                                    --container-name="generic-metrics-distributions-consumer" \
-                                    --container-name="generic-metrics-distributions-subscriptions-executor" \
-                                    --container-name="generic-metrics-distributions-subscriptions-scheduler" \
-                                    --container-name="generic-metrics-sets-consumer" \
-                                    --container-name="generic-metrics-sets-subscriptions-executor" \
-                                    --container-name="generic-metrics-sets-subscriptions-scheduler" \
-                                    --container-name="loadbalancer-outcomes-consumer" \
-                                    --container-name="metrics-consumer" \
-                                    --container-name="metrics-counters-subscriptions-scheduler" \
-                                    --container-name="metrics-sets-subscriptions-scheduler" \
-                                    --container-name="metrics-subscriptions-executor" \
-                                    --container-name="outcomes-billing-consumer" \
-                                    --container-name="outcomes-consumer" \
-                                    --container-name="querylog-consumer" \
-                                    --container-name="replacer" \
-                                    --container-name="snuba-admin" \
-                                    --container-name="transactions-consumer-new" \
-                                    --container-name="transactions-subscriptions-executor" \
-                                    --container-name="transactions-subscriptions-scheduler" \
+                                    --container-name=""api" \
```

Gotta fix all of these:

```suggestion
                                    --container-name="api" \
```